setup.cfg

```diff
@@ -37,7 +37,7 @@ install_requires =
     pilkit
 
 [options.extras_require]
-all = boto; brotli; feedgenerator; zopfli
+all = boto; brotli; feedgenerator; zopfli; cryptography; beautifulsoup4
```

Comment:
Since JS files are now injected via Jinja templates, `beautifulsoup4` is no longer a dependency.
Note to self: remove bs4 from code.

Reply:
Removed in 28f187f.